### PR TITLE
Includes pricing data in review, and shows the data in appropriate places

### PR DESCRIPTION
**Pull request by shaneponcik**

### CODE REVIEW

File: app/models/apartment.rb [hunk 1]
```diff
@@ -39,4 +39,49 @@ def get_tags
...
+
+    sum = 0
+    reviews.each do |review|
+      sum = sum + review.price
+    end
+    return sum/reviews.size
+  end
+
+  def get_min_price
+    reviews = Review.where(:apartment_id => self.id)
```
Comment (JHannTX) [inline app/models/apartment.rb]: Isn't there a min query function?

File: db/seeds.rb [hunk 2]
```diff
@@ -45,27 +45,6 @@
...
-            [
-                {review_id:1,amenity_id:1},
-                {review_id:1,amenity_id:4}])
-        review_metrics = ReviewMetric.create(
-            [
-                {review_id:1,metric_id:1,rating:5},
-                {review_id:1,metric_id:2,rating: 2},
-                {review_id:2,metric_id:3,rating: 4}
-            ])
-
```
Comment (JHannTX) [inline db/seeds.rb:68]: Why was this removed?

**Comment by JHannTX** [comment]
Just making sure this isn't supposed to have complete testing yet and the changes to the previous testing are because the changes broke those tests.

File: app/models/apartment.rb (hunk 1 above)
Comment (JHannTX) [reply app/models/apartment.rb]: There are queries that have this functionality.  Here is a link that talks about them.
https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html

File: db/seeds.rb (hunk 2 above)
Comment (shaneponcik) [reply db/seeds.rb:68]: reviews were incomplete, so i just removed them because we have "add review" functionality

File: app/models/apartment.rb (hunk 1 above)
Comment (shaneponcik) [reply app/models/apartment.rb]: i'll touch that up in a later submission

**Review by ColbyMainard — CHANGES_REQUESTED** [review]
I think we should make changes we can see will be required to this code BEFORE officially merging

File: app/models/apartment.rb (hunk 1 above)
Comment (ColbyMainard) [reply app/models/apartment.rb]: Just put in a change request.  Once I know the changes Julia mentioned have been properly addressed, I will approve the request.

**Comment by shaneponcik** [comment]
re-creating with changes
